PDSA_System/Server/Controllers/BrukerController.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using PDSA_System.Server.Models;
+using Dapper;
+
+// For more information on enabling MVC for empty projects, visit https://go.microsoft.com/fwlink/?LinkID=397860
+
+namespace PDSA_System.Server.Controllers
+{
+    [Route("[controller]")]
```

Comment:
Kan det være en ide å gjemme slikt bak `/api`? Da får man et mer tydelig skille mellom client og server-side

```suggestion
    [Route("/api/[controller]")]
```

Reply:
enig 👍